frontend/src/components/pages/rp-connect/secrets/Secrets.Create.tsx

```diff
@@ -0,0 +1,142 @@
+import {Button, ButtonGroup, createStandaloneToast, Flex, FormField, Input, PasswordInput} from '@redpanda-data/ui';
+import {PageComponent, PageInitHelper} from '../../Page';
+import {observer} from 'mobx-react';
+import {appGlobal} from '../../../../state/appGlobal';
+import {pipelinesApi, rpcnSecretManagerApi} from '../../../../state/backendApi';
+import PageContent from '../../../misc/PageContent';
+import {action, makeObservable, observable} from 'mobx';
+import {DefaultSkeleton} from '../../../../utils/tsxUtils';
+import {formatPipelineError} from '../errors';
+import {CreateSecretRequest, Scope} from '../../../../protogen/redpanda/api/dataplane/v1alpha2/secret_pb';
+
+const {ToastContainer, toast} = createStandaloneToast();
+
+const returnSecretTab = '/connect-clusters?defaultTab=redpanda-connect-secret'
+
+
+@observer
+class RpConnectSecretCreate extends PageComponent {
+
+    @observable id = '';
+    @observable secret = '';
+    @observable isCreating = false;
+
+    constructor(p: any) {
+        super(p);
+        makeObservable(this, undefined, {autoBind: true});
+    }
+
+    initPage(p: PageInitHelper) {
+        p.title = 'Create Secret';
+        p.addBreadcrumb('Redpanda Connect Secret Manager', '/rp-connect/secret/create');
+        p.addBreadcrumb('Create Secret', '');
+
+        this.refreshData(true);
+        appGlobal.onRefresh = () => this.refreshData(true);
+    }
+
+    refreshData(_force: boolean) {
```

Comment:
very nit: maybe we could keep the convention and just use `force` to be consistent with other pages